Tests.Unit.Presentation.Web/Models/V2/ItContractResponseMapperTest.cs

```diff
@@ -75,6 +75,7 @@ public void MapContractDTO_Maps_No_Properties()
             Assert.Null(general.ContractType);
             Assert.Null(general.ContractTemplate);
             Assert.Empty(general.AgreementElements);
+            Assert.Null(general.CriticalityType);
```

Comment:
you are only testing the null case (empty mapping). That's not enough. There are explicit tests for both procurement and general